### DISCUSSION TITLE
Ongoing updates

### DISCUSSION THREAD

**Discussion by robindhillon1** [UBC-MDS/chocolate_rating#58] [Ideas]
Reminder to add explicitly what the report is named and where to find in the README. Even better to link directly to it. 

If there are any other updates/changes that need to be done, add them here as we proceed.

**Comment by robindhillon1** [comment]
-  The following feedback from Milestone 1 has already been taken care of: 
    - The question(s) should be specific enough. 
    - Clarity of README could be improved through additional proofreading (i.e., spelling and grammatical errors are present).
    - Added `docopt` to dependencies.
    - `kableExtra` added in `README` per Milestone 2 feedback.
- Currently working on:
    - Code of conduct update from Milestone 1.
    - Contributing file update from Milestone 1.
    - Adding titles, labels, and captions for figures and tables.

**Reply by markusnam** [reply]
DONE:
- Add tests for all the scripts.
- Reduce number of options for `model_summary.py` (Alex's comment)
- Update Makefile accordingly

**Reply by markusnam** [reply]
@robindhillon1 
In README.md, please update the second last line of `Usage` (for `model_summary.py`)
FROM
`python src/model_summary.py --in_file=data/processed/test.csv --model_dir=results/ --dummy=model_baseline.sav --svr=model_svr.sav --ridge=model_ridge.sav --out_dir=results/`

TO
`python src/model_summary.py --in_file=data/processed/test.csv --model_dir=results/ --out_dir=results/`

**Reply by markusnam** [reply]
# Summary as at 15:15 Dec 7
Improvements done:
- [Peer Review] Add tests for all the scripts
- [Peer Review] Reduce number of options for model_summary.py
- [Peer Review] Rounding for coefficients and MAPE scores.
- [Peer Review] Add a paragraph on the choice of using MAPE.
- [Milestone 1] The question(s) should be specific enough.
- [Milestone 1] Clarity of README could be improved through additional proofreading (i.e., spelling and grammatical errors are present).
- [Milestone 1] Add docopt to dependencies.
- [Milestone 2] Address the kableExtra issue from milestone 2 feedback.
- [Milestone 2] Address the pandoc not found issue.

**Comment by eyrexh** [comment]
- Uniformed the rounding to be all 3 decimals.
- Added the MAPE explanation
